eng/targets/Helix.targets

```diff
@@ -28,6 +28,15 @@
   <ItemGroup Condition="$(BuildHelixPayload)">
     <Content Include="@(HelixContent)" />
     <Content Include="$(RepoRoot)eng\scripts\Download.ps1" />
+    <Content Include="$(RepoRoot)NuGet.config" />
```

Comment:
I realized after doing this that we probably don't need to add as many sources in eng/helix/content/RunTests/TestRunner.cs anymore. That file also adds a bunch of environment variables that are set in the `runtests.*` scripts. Please file a follow-up issue for cleanup after this goes in. Or, just open another PR to remove the redundancy.

I'm keeping this separate because it's not really part of the merge.

Reply:
/cc @JunTaoLuo

Reply:
Plz file follow-up issue(s) to
1. Remove `TestRunner` NuGet source additions that should already be in this file (copied from the build and, likely enhanced in the case of internal builds)
2. Reduce the duplication between `TestRunner` and the `runtests.*` scripts if possible
  - might be more there than just environment variables
3. Add the new `dotnet-install.*` scripts from eng/common/dotnet-install-scripts/ to the Helix payload (placing them in a dotnet-install-scripts/ folder relative to where the `tools.*` files land